Explorer/Assets/DCL/FeatureFlags/FeaturesRegistry.cs

```diff
@@ -100,6 +100,14 @@ public void RegisterFeatureProvider(FeatureId featureId, IFeatureProvider provid
     public enum FeatureId
     {
         NONE = 0,
+        VOICE_CHAT = 1,
+        COMMUNITY_VOICE_CHAT = 2,
+        FRIENDS = 3,
+        FRIENDS_USER_BLOCKING = 4,
+        FRIENDS_ONLINE_STATUS = 5,
+        PROFILE_NAME_EDITOR = 6,
+        LOCAL_SCENE_DEVELOPMENT = 7,
+        CAMERA_REEL = 8,
```

Comment:
make sure that this didn't mess up any settings feature flag bindings as changing the order can mess them up